README.md

```diff
@@ -18,10 +18,18 @@ Nominatim, or general analysis.
 
 ## Installing ##
 
-Most Linux distributions include osm2pgsql. It is also available on macOS with [Homebrew](http://brew.sh/).
+**Linux**: Most distributions include osm2pgsql.
 
-Unoffical builds for Windows are available from [AppVeyor](https://ci.appveyor.com/project/openstreetmap/osm2pgsql/history) but you need to find the right build artifacts. The latest
-release is [0.92.0](https://ci.appveyor.com/api/projects/openstreetmap/osm2pgsql/artifacts/osm2pgsql_Release.zip?tag=0.92.0).
+**macOS**: osm2pgsql is available with [Homebrew](http://brew.sh/).
+
+**Windows**: Unoffical builds are available from [AppVeyor](https://ci.appveyor.com/project/openstreetmap/osm2pgsql/history). The latest
+release is [0.94.0](https://ci.appveyor.com/api/projects/openstreetmap/osm2pgsql/artifacts/osm2pgsql_Release.zip?tag=0.94.0). Just unzip the archive into a directory of your choice. You may need Admnistrator permissions in order to unzip into `C:\Program Files (x86)`.
```

Comment:
The sentence "You may need Admnistrator […]" is not necessary. That's obvious. Users who don't know these very basics of permission management on Windows are often also unable to use the command line and therefore unable to use Osm2pgsql.

Reply:
Windows is not the primary target of the osm2pgs distribution. For example, it does not have an installer as other Windows software packages have. It would be nice to help Windows users in overcoming that hurdle.